test/uk/gov/hmrc/plasticpackagingtax/registration/views/ConfirmationViewSpec.scala

```diff
@@ -38,14 +39,15 @@ class ConfirmationViewSpec extends UnitViewSpec with Matchers {
 
       messages must haveTranslationFor("confirmationPage.title")
       messages must haveTranslationFor("confirmationPage.panel.body")
-      messages must haveTranslationFor("confirmationPage.body")
+      messages must haveTranslationFor("confirmationPage.panel.body.default")
+      messages must haveTranslationFor("confirmationPage.detail.1")
+      messages must haveTranslationFor("confirmationPage.detail.2")
       messages must haveTranslationFor("confirmationPage.whatHappensNext.title")
-      messages must haveTranslationFor("confirmationPage.whatHappensNext.paragraph1")
-      messages must haveTranslationFor("confirmationPage.whatHappensNext.liable.title")
-      messages must haveTranslationFor("confirmationPage.whatHappensNext.liable.bulletPoint.1")
-      messages must haveTranslationFor("confirmationPage.whatHappensNext.liable.bulletPoint.2")
-      messages must haveTranslationFor("confirmationPage.exitSurvey.text.link")
-      messages must haveTranslationFor("confirmationPage.exitSurvey.text")
+      messages must haveTranslationFor("confirmationPage.whatHappensNext.detail")
+      messages must haveTranslationFor("confirmationPage.whatHappensNext.link")
+      messages must haveTranslationFor("confirmationPage.whatHappensNext.link.text")
+      messages must haveTranslationFor("confirmationPage.exitSurvey.link")
+      messages must haveTranslationFor("confirmationPage.exitSurvey.link.text")
```

Comment:
Just a reminder - since we moved to the "validating" messages api in the view tests, we don't need to assert each message.  If it is used in the view it will fail if the translation is not present.